### DISCUSSION TITLE
Simpled Smart Lock

### DISCUSSION THREAD

**Discussion by Jonnehs** [jbergler/hass-ttlock#147] [General]
Does anyone know if this works with the Simpled Hub that they supply with their locks, or if you need to buy the generic TTLock hub?

Thanks

**Comment by jbergler** [comment]
To the best of my knowledge, the ttlock ecosystem uses the same backend for all vendors.
You should be able to install the unbranded ttlock app and login with the same credentials. If that works, then the integration will too.

**Reply by Jonnehs** [reply]
grateful for your response, appreciated
